### PR TITLE
Most-travelled-destinations-181414583

### PR DESCRIPTION
**Pull request by iyaremye-faustin**
**1. Name of the Task**

Most traveled Destinations

**2. Pivot Track ID**

181414583
**3. Branch Name**

ft-most-travelled-destinations-#181414583

**4 Task Description**

Given that travelers choose different destinations, the user should be given the most traveled destinations to help make the choice.

** 5 How can be it tested**

Given that the user is logged in and has accessed the dashboard he can the link to the top destinations and should be logged in as a Manager or Requester and there should be some trip requests already booked

### CODE REVIEW

File: src/views/pages/HomeUserView.js [hunk 1]
```diff
@@ -4,21 +4,20 @@ import DashboardHeader from '../../components/Header/DashboardHeader';
...
-    <div>
-      <div className="flex">
-        <SideBar />
-        <div className="w-full mx-[2vw] md:ml-[21vw] overflow-y-auto">
-          <DashboardHeader />
-          {page === 'trips' ? <Trip /> : <div />}
-        </div>
-      </div>
-      <div className="md:hidden">
-        <MobileMenu />
```
Comment (rukundo-kevin) [inline src/views/pages/HomeUserView.js]: Why did you remove MobileMenu

**Review by rukundo-kevin — CHANGES_REQUESTED** [review]

File: src/views/pages/HomeUserView.js (hunk 1 above)
Comment (rukundo-kevin) [inline src/views/pages/HomeUserView.js]: Why did you remove MobileMenu

**Review by rukundo-kevin — CHANGES_REQUESTED** [review]
Add most travelled destination for one accommodation

File: src/views/pages/HomeUserView.js (hunk 1 above)
Comment (EmyRukundo) [reply src/views/pages/HomeUserView.js]: @iyaremye-faustin

**Comment by EmyRukundo** [comment]
Fix the code climate issues

**Review by EmyRukundo — CHANGES_REQUESTED** [review]

**Comment by rukundo-kevin** [comment]
Rebase

**Review by NNesta — DISMISSED** [review]
LGTM

**Review by rukundo-kevin — DISMISSED** [review]

**Review by mucyoalexaxel — APPROVED** [review]
Good Work @iyaremye-faustin

**Pull request merged**